src/LaunchDarkly.EventSource/EventSource.cs

```diff
@@ -135,7 +135,7 @@ public EventSource(Configuration configuration)
             if (_backOff.GetReconnectAttemptCount() > 0 && _retryDelay > TimeSpan.FromMilliseconds(0))
             {
                 TimeSpan sleepTime = _backOff.GetNextBackOff();
-                _logger.LogInformation("Waiting {0} milliseconds before reconnecting...", sleepTime.TotalMilliseconds);
+                _logger.Info(String.Format("Waiting {0} milliseconds before reconnecting...", sleepTime.TotalMilliseconds));
```

Comment:
Can we use `InfoFormat` (etc.) instead? In general it seems like good practice to avoid unnecessary string formatting for messages that might not ever be output.